actions/setup/action.yml

```diff
@@ -16,6 +16,10 @@ inputs:
     description: Number of commits to fetch. 0 indicates all history for all branches and tags.
     default: 0
 
+  git-branch:
+    description: What branch to checkout
+    default: master
```

Comment:
I guess this introduces a hard dependency on master as default branch (if that requirement did not exist before).

Reply:
That's OK IMO - we should have this standardized